int16 fwd_pressure
int16 aft_pressure
int16 temp
